packages/bigtable/src/filter.js

```diff
@@ -170,7 +170,7 @@ Filter.createRange = function(start, end, key) {
 
   function createBound(boundName, boundData, key) {
     var isInclusive = boundData.inclusive !== false;
-    var boundKey = boundName + key + (isInclusive ? 'Inclusive' : 'Exclusive');
+    var boundKey = boundName + key + (isInclusive ? 'Closed' : 'Open');
```

Comment:
Well, the docs within the proto files still refer to open and closed being either inclusive or exclusive. I originally decided against changing the terminology because I felt the original verbiage was more intuitive.

WDYT?